docs/categories.yml

```diff
@@ -62,6 +62,7 @@ starter:
   - Styling:None
   - Testing
   - Transitions
+  - Userbase
```

Comment:
We do not allow for editing on this file to add categories. Can you revert this?

Reply:
Ah okay, sorry about that, in the submit a starter documentation it mentioned you could submit a pull request to add a category. Has that changed then since the documentation has been published? Is there any chance this can be added? If so, how can I go about that process? Thank you!